addons/l10n_fr_account/models/account_move.py

```diff
@@ -19,3 +19,17 @@ def _get_view(self, view_id=None, view_type='form', **options):
     def _compute_l10n_fr_is_company_french(self):
         for record in self:
             record.l10n_fr_is_company_french = record.country_code in record.company_id._get_france_country_codes()
+
+    @api.depends('country_code', 'move_type')
+    def _compute_show_delivery_date(self):
+        # EXTENDS 'account'
+        super()._compute_show_delivery_date()
+        for move in self:
+            if move.country_code == 'FR':
```

Comment:
Instead of doing that, look why we have is_company_french, maybe you could use it instead :smile: